apps/beeswax/src/beeswax/server/hive_server2_lib.py

```diff
@@ -379,6 +379,7 @@ def set_nulls(cls, values, bytestring):
 
 class HiveServerDataTable(DataTable):
   def __init__(self, results, schema, operation_handle, query_server, session=None):
+    self.results = results
```

Comment:
I don't think we are using this anywhere, good to remove?

Reply:
to get the results this line was necessary - 
tables = result.data_table.results.results.columns[0].stringVal.values

Reply:
@divincode - I think we can use `self.row_set` to get `results.results` instead of using `self.results`?
You can check the `HiveServerTRowSet` class for reference.